x-pack/plugins/security_solution/public/resolver/view/panels/related_event_detail.tsx

```diff
@@ -138,9 +139,9 @@ export const RelatedEventDetail = memo(function RelatedEventDetail({
     }
   }, [relatedsReady, dispatch, processEntityId]);
 
-  const relatedEventsForThisProcess = useSelector(selectors.relatedEventsByEntityId).get(
-    processEntityId!
-  );
+  const relatedEventsForThisProcess = useShallowEqualSelector(
```

Comment:
Also unrelated to this PR, but since I'm checking the semantics of stuff, this should be a parametric selector also.